apps/api/src/Controller/DefaultController.php

```diff
@@ -376,6 +408,47 @@ private function doGreencheck($url, $ip, $browser, $source = 'api', $blind = fal
         return $data['result'];
     }
 
+     /**
+     * @param $url
+     * @param $ip
+     * @param $browser
+     * @param string $source
+     * @param bool   $blind
+     *
+     * @return mixed
+     */
+    private function doGreencheckAsync($url, $ip, $browser, $source = 'api', $blind = false)
+    {
+        $promise = $this->producer->sendCommand('greencheck', JSON::encode(['key' => 0, 'url' => $url, 'ip' => $ip, 'browser' => $browser, 'source' => $source, 'blind' => $blind]), $needReply = false);
```

Comment:
no need to assign to a $promise as it won't return anything and we won't use the varaible later :)